fabfile.py

```diff
@@ -228,6 +227,27 @@ def postgres_load_database(backup_name, new_env):
     ))
 
 
+def write_cron_backup(new_env):
```

Comment:
@fredkingham This function has just moved and had a docstring line added?

Reply:
yes its exactly the same.